templates/account.php

```diff
@@ -243,6 +243,10 @@
     $addons_to_show = array_unique( array_merge( $installed_addons_ids, $account_addons ) );
 
     $is_active_bundle_subscription = ( is_object( $bundle_subscription ) && $bundle_subscription->is_active() );
+
+    $available_license = $fs->is_free_plan() && ! fs_is_network_admin() ? $fs->_get_available_premium_license($site->is_localhost()) : false;

```

Comment:
@swashata let's change the formatting so it will be easier to digest:
```
$available_license = ( $fs->is_free_plan() && ! fs_is_network_admin() ) ?
    $fs->_get_available_premium_license( $site->is_localhost() ) :
    false;
```